.github/workflows/ci.yml

```diff
@@ -14,6 +14,7 @@ jobs:
         with:
           cache: 'yarn'
       - run: yarn install --frozen-lockfile
+      - run: yarn build
```

Comment:
does this mean we need to build the lint packages before linting?

Reply:
i've just tested with a
`yarn clean`
and a
`yarn lint`
and it seems that doing a build doesn't seem to be necessary for our lint process

Reply:
I have removed it in my latest push

Reply:
ah — nevermind, I needed to build both of the dogfooded eslint plugins. as the others are unnecessary I pushed a change to the ci step that builds just those two:

```yml
 # Build just the two eslint plugins needed.
      - run: yarn nx run-many --target=build --projects='eslint-plugin-source-foundations,eslint-plugin-source-react-components'
```